### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by NullYing**
```
Error: caskroom/cask was moved. Tap homebrew/cask-cask instead.
```

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ Installation
 
 .. code-block:: bash
 
-	$ brew install caskroom/cask/wkhtmltopdf
+	$ brew install homebrew/cask-cask/wkhtmltopdf
```
Comment (Lucurus) [inline README.rst:36]:
```suggestion
	$ brew cask install wkhtmltopdf
```

As outlined in https://github.com/JazzCore/python-pdfkit/issues/149#issuecomment-677507216 brew specifies the correct method as `brew cask install`.



I have also created #168 as I hadn't seen this one but only one is necessary so I can close mine as well.

**Review by Lucurus — CHANGES_REQUESTED** [review]
I would suggest changing the brew cask command.

**Comment by ArtemBernatskyy** [comment]
it seems that this repo has died, xD

**Comment by BastianZim** [comment]
Yeah, unfortunately.

**Comment by JazzCore** [comment]
``brew cask install wkhtmltopdf`` does not work for me on current MacOS, but ``brew install homebrew/cask/wkhtmltopdf`` does, fixed in #173, thanks guys and sorry that it took so long to get to it.

**Pull request closed without merging**